templates/archives/stories/meesue_story.jinja2

```diff
@@ -0,0 +1,57 @@
+{% extends 'archives/story.jinja2' %}
+
+{% block story_title %}
+The Rise and Fall of Project Whirlwind
+{% endblock %}
+
+{% block story_credit %}
+
+{% endblock %}
+
+
+{% block story_body %}
+<p>
+    Arguably the first large-scale and high-speed computer, the development of the Whirlwind Project started in 1944 by
+    the MIT Servomechanisms Laboratory in a project directed by Jay Forrester for the United States Navy. This project
```

Comment:
The grammar on this is a little off as the computer is not referred to by name but by project. I would re-phrase:

```
Arguably the first large-scale and high-speed computer, the Whirlwind was developed by the MIT Servomechanisms Laboratory in a project directed by Jay Forrester for the United States Navy.
```

You can add the fact about the date into another sentence.